src/ComposerPlugin.php

```diff
@@ -8,6 +8,10 @@
  *
  */
 
+/**************************************/
+/***** THIS FILE IS PHP, NOT HACK *****/
```

Comment:
Will this always have to exist? What about when HHVM doesn't support PHP?

Reply:
I think it's still need on some LTS.

We need it on releases that refuse to run Hack code with typechecker errors.